__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+const connection = require('../db/connection');
+const request = require('supertest');
+const endpoints = require('../endpoints.json')
+
+afterAll(() => {
+    return connection.end();
+})
+
+beforeEach(() => {
+    return seed(data);
+})
+
+describe("/api/topics", () => {
+    test("200: response with a status of 200", () => {
+        return request(app).get("/api/topics").expect(200)
+    });
+    test('"200: responds with an array of topic objects"', () => {
+        return request(app).get("/api/topics")
+        .then(({ body }) => {
+            const topics = body.topics.rows
+            console.log(topics);
+            expect(Array.isArray(topics)).toBe(true)
+            expect(topics.length).toBe(topics.length)
+            topics.forEach((topic) => {
+                expect(topic).toHaveProperty('slug')
+                expect(topic).toHaveProperty('description')
+            })
+        })
+    });
```

Comment:
Nice destructuring here. 
topics.length will always be topics.length -  make sure you use the actual length of the test data here as a number.
Nice testing of each object's contents here. 
Don't forget to remove console.logs.